bridge-fungible-token/tests/utils/environment.rs

```diff
@@ -15,53 +15,53 @@ use fuels::{
     prelude::*,
     types::{message::Message, Bits256},
 };
-use primitive_types::U256;
+use primitive_types::U256 as Unsigned256;
```

Comment:
Why did you add an alias?

Reply:
The Rust `U256` was conflicting with the Sway `U256` name found in the Contract module.